documentation/jetty/modules/programming-guide/pages/troubleshooting/jetty-server-dump.adoc

```diff
@@ -0,0 +1,116 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+[[dump]]
+= Jetty Server Dump
+
+Jetty is based on components organized as a tree, with the `Server` instance at the root of the tree.
+
+As explained in the xref:operations-guide:jmx/index.adoc[JMX section], these components can be exported as JMX MBeans and therefore be accessible from JMX Consoles such as Java Missions Control (JMC).
+
+Being able to take a snapshot of the state of Jetty while it is running is the most useful information that can be attached when reporting an issue.
```

Comment:
```suggestion
Being able to take a snapshot of a Jetty component tree while it is running is useful information that can be attached when reporting an issue as it can contain both configuration and current state of Jetty.
```